tests/backup/backup_test.go

```diff
@@ -1303,6 +1325,131 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+				time.Sleep(time.Minute * 4)
+				CreateBackupLocation(provider, bkpLocationName, backupLocationUID, cloudCredName, cloudCredUID, bucketName, orgID, "")
+			}
+		})
+		Step("Register cluster for backup", func() {
+			CreateSourceAndDestClusters(orgID, "", "", ctx)
+			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, "Verifying backup cluster")
+		})
+		Step("Taking backup of applications", func() {
+			backupName = fmt.Sprintf("%s-%s", BackupNamePrefix, bkpNamespaces[0])
+			err = CreateBackup(backupName, SourceClusterName, bkpLocationName, backupLocationUID, []string{bkpNamespaces[0]},
+				labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+			dash.VerifyFatal(err, nil, "Verifying backup creation")
+		})
+		Step("Share backup with  user having full access", func() {
+			log.InfoD("Share backup with  user having full access")
+			ShareBackup(backupName, nil, []string{userName}, FullAccess, ctx)
+		})
+		Step("Create  backup from the shared user with FullAccess", func() {
+			log.InfoD(" Validating if user with  FullAccess cannot duplicate backup shared but can create new backup")
+			// User with FullAccess cannot duplicate will be validated through UI only
+			ctxNonAdmin, err = backup.GetNonAdminCtx(userName, "Password1")
+			log.FailOnError(err, "Fetching user ctx")
+			log.InfoD("Registering Source and Destination clusters from user context")
+			CreateSourceAndDestClusters(orgID, "", "", ctxNonAdmin)
+			userBackupName = fmt.Sprintf("%s-%s-%s", "user", BackupNamePrefix, bkpNamespaces[0])
+			err = CreateBackup(userBackupName, SourceClusterName, bkpLocationName, backupLocationUID, []string{bkpNamespaces[0]},
+				labelSelectors, orgID, clusterUid, "", "", "", "", ctxNonAdmin)
+			dash.VerifyFatal(err, nil, "Verifying that create backup should pass ")
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
+		ValidateAndDestroy(contexts, opts)
+		log.Infof(" Deleting backup created by px-central-admin")
+		backupDriver := Inst().Backup
+		backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+		dash.VerifySafely(err, nil, "Getting backup UID")
+		DeleteBackup(backupName, backupUID, orgID, ctx)
+		time.Sleep(time.Minute * 3)
+		log.Infof(" Deleting backup created by user")
+		userBackupUID, err := backupDriver.GetBackupUID(ctxNonAdmin, userBackupName, orgID)
+		dash.VerifySafely(err, nil, "Getting backup UID of user")
+		DeleteBackup(userBackupName, userBackupUID, orgID, ctxNonAdmin)
+		time.Sleep(time.Minute * 3)
+		log.Infof("Cleaning up users")
+		err = backup.DeleteUser(userName)
+		log.FailOnError(err, "Error in deleting user")
+		log.Infof("Deleting registered clusters for non-admin context")
```

Comment:
Can we use `DeleteCloudAccounts` here?

Reply:
DeleteCloudAccounts deletes backup location and cloud cred also.
for user we do not have that, hence used individual method

Reply:
Okay